docs/index.html

```diff
@@ -0,0 +1,30 @@
+<html>
+  <head>
+    <script src="Chart.bundle.js"></script>
+    <script src="moment.js"></script>
```

Comment:
do we need to check this file in? maybe we should reference from a CDN like in samples/ohlc.html

Reply:
If the file is checked in, once the repository is updated, the samples are working anywhere, anytime, even offline.